examples/asr/experimental/wpe_vocabs/1k/vocab.txt

```diff
@@ -0,0 +1,1024 @@
+[PAD]
```

Comment:
Do we need these files?

Reply:
Other people can not train this model without these vocab files. These files are not publicly available.

Reply:
@VahidooX these files are not really needed. When we train the EncDecCTCBPEModel, the underlying model will have the vocab file attached to it in our checkpoints. So these vocabs dont really need to be present inside NeMo. And I believe Vocab construction using the script is deterministic as well.

Reply:
I see this file was removed. Resolving this conversation.